drivers/digital-io/max22196/iio_max22196.c

```diff
@@ -301,20 +301,7 @@ static struct iio_attribute max22196_debug_attrs[] = {
 		.attributes = max22196_attrs	\
 	}
 
-static struct iio_channel max22196_channels[] = {
-	MAX22196_CHANNEL(0),
-	MAX22196_CHANNEL(1),
-	MAX22196_CHANNEL(2),
-	MAX22196_CHANNEL(3),
```

Comment:
is this change related to max22194 support?

Reply:
Yes, channels in the iio driver will now be allocated dynamic according to the chip id